MIGRATE.md

```diff
@@ -52,16 +52,17 @@ server or note its path if the same server will be used.
 
 1. Make sure your rmt installation is up-to-date. `rmt-data-import` is available in RMT versions >= 1.0.0
 2. Unpack the tarball containing SMT data to some directory, e.g. `/root/smt-data`
-3. If you chose to export SMT's SSL certificates, decrypt the SMT CA private key to `/usr/share/rmt/ssl/`:
+3. If you chose to export SMT's SSL certificates, copy the SMT CA private key to `/usr/share/rmt/ssl/`:
     ```
-    openssl rsa -in /root/smt-data/ssl/cacert.key -out /usr/share/rmt/ssl/rmt-ca.key
+    cp /root/smt-data/ssl/cacert.key /usr/share/rmt/ssl/rmt-ca.key
     ```
+    It is strongly recommended to encrypt your CA private key.
```

Comment:
@ikapelyukhin @Sergeykot  Is this line needed? Does SMT only deal with encrypted SSL certificates?

Reply:
well the non-encrypted certificate will still work (in YaST2-RMT).
I would leave it since this is a security recommendation

Reply:
however I can remove it since it was not present before in the doc

Reply:
On SMT it's always encrypted.

Reply:
Then I would remove the line.

Reply:
fixed